docs/source/openvino/export.mdx

```diff
@@ -31,7 +31,7 @@ Check out the help for more options:
 
 ```text
 usage: optimum-cli export openvino [-h] -m MODEL [--task TASK] [--framework {pt,tf}] [--trust-remote-code]
-                                   [--weight-format {fp32,fp16,int8,int4,mxfp4,nf4}]
+                                   [--weight-format {fp32,fp16,int8,int4,mxfp4,nf4,cb4_f8e4m3}]
```

Comment:
may be better to call `f8e4m3`?

Reply:
Another suggestion to use `cb4` for weight-format